main.go

```diff
@@ -46,34 +47,65 @@ import (
 func main() {
 	LogLady.Info("Starting Nancy")
 
+	var err error
 	if len(os.Args) > 1 && os.Args[1] == "iq" {
-		LogLady.Info("Nancy parsing config for IQ")
-		config, err := configuration.ParseIQ(os.Args[2:])
-		if err != nil {
-			flag.Usage()
-			os.Exit(1)
-		}
-		LogLady.WithField("config", config).Info("Obtained IQ config")
-		processIQConfig(config)
-		LogLady.Info("Nancy finished parsing config for IQ")
+		err = doIq(os.Args[2:])
```

Comment:
I like that you simplified the func name, but now it's a lil impossible to tell what it actually does. `doIq` is a bit too simple, I think. As well, it should likely be `doIQ`, as general golang convention is to capitalize stuff like that (`URL` vs `Url`, etc...)

Reply:
These `do*` methods will likely map directly to new Cobra `command` structures. I will try to come up with better names for the Cobra commands. For now, the main goal was to simplify the main method enough so I could grok replacing the `os.Exit` stuff with centralized "exit processing" in `main()`.